lib/srv/desktop/tdp/proto.go

```diff
@@ -513,14 +545,19 @@ func (c ClipboardData) Encode() ([]byte, error) {
 	return buf.Bytes(), nil
 }
 
+const clipDataMaxLenErr = "clipboard sync failed: clipboard data exceeded maximum length"
```

Comment:
If you take my `trace.IsLimitExceeded` advice above, you can inline these messages and don't have the need for separate constants.

Also, two minor nits:

- the `Err` suffix on the variable name suggest this is an `error` but it's actually a string
- You have some inconsistent naming - here the suffix is `Err` but below you have mfaDataMaxLenError` with `Error` as a suffix